### PR TITLE
Update make.bat

### PR DESCRIPTION
**Pull request by Atharva1723**
I've updated the URL in the error message to provide a more specific and up-to-date link to the Sphinx documentation related to installation. The new URL points directly to the "Installation" section of the documentation, making it easier for users to find the information they need to resolve the issue.

**Why this change is necessary**:
The current URL points to the root of the Sphinx documentation website, which is less specific and might not provide immediate guidance for users facing an installation issue. The updated URL ensures that users are directed to the correct section, improving their experience and reducing the chances of confusion.

**The new URL uses "https://" for a secure connection.**
This change is a minor improvement that enhances the user experience when encountering installation issues with Sphinx.

### CODE REVIEW

**Comment by Atharva1723** [comment]
@ryshu Please review this PR! Thank You :)

**Comment by Atharva1723** [comment]
@ryshu Can you Please review this PR ? Thank You :)

**Review by ryshu — CHANGES_REQUESTED** [review]
Hi,

For me there is no need to reinvent the wheel on this file because it is autogenerated.

If you find that the documentation is not adequate, you might as well submit an issue directly to the Sphinx repository so that their codes can be updated: https://github.com/sphinx-doc/sphinx

From my point of view, there is a very good reason for not putting a full URL here: if the documentation link changes, the previous code is still good while the proposed code is no longer good.

The https has since been corrected on their repositories in fact, we can correct it here, but for me the modification must be a copy and paste of this file: https://github.com/sphinx-doc/sphinx/blob/master/doc/make.bat

**Comment by Atharva1723** [comment]
> Hi,
> 
> For me there is no need to reinvent the wheel on this file because it is autogenerated.
> 
> If you find that the documentation is not adequate, you might as well submit an issue directly to the Sphinx repository so that their codes can be updated: https://github.com/sphinx-doc/sphinx
> 
> From my point of view, there is a very good reason for not putting a full URL here: if the documentation link changes, the previous code is still good while the proposed code is no longer good.
> 
> The https has since been corrected on their repositories in fact, we can correct it here, but for me the modification must be a copy and paste of this file: https://github.com/sphinx-doc/sphinx/blob/master/doc/make.bat

Yes, it is better to include the relative url  as it ensures that the link will still work within the context of the file, even if the actual documentation location changes.

**Comment by unicodeveloper** [comment]
So can we close this PR now?

**Comment by Atharva1723** [comment]
> So can we close this PR now?

Actually i have made an fresh commit that updates the bat file according to the sphinx original file

**Comment by unicodeveloper** [comment]
We appreciate your efforts. However, there are other issues that are priority that you can handle. Updating this right now does not provide the users of this SDK any value. So, please can you look into the issues, we have things that can be documented and features that can be added to the SDK. Sorry about this @Atharva1723

File: docs/make.bat [hunk 1]
```diff
@@ -1,19 +1,16 @@
-:: This file is covered by the LICENSE file in the root of this project.
 @ECHO OFF
 
-pushd %~dp0
-
 REM Command file for Sphinx documentation
 
 if "%SPHINXBUILD%" == "" (
-	set SPHINXBUILD=sphinx-build
+	set SPHINXBUILD=python ../sphinx/cmd/build.py
```
Comment (ryshu) [inline docs/make.bat:6]: I think this line only work on sphinx repositiory

```suggestion
	set SPHINXBUILD=sphinx-build

```

**Review by ryshu — APPROVED** [review]

**Pull request merged**